Tutorial/GettingStartedGuide.md

```diff
@@ -0,0 +1,148 @@
+# Getting Started Guide  
+  
+This guide explains how to package up your trained model as a web service with a predict API that can be scanned using Certifai
+  
+  
+## Table of Contents  
+1. [pre-requisites](#pre-requisites)   
+2. [predict](#predict)  
+3. [api](#api)  
+4. [build-container](#build-container)  
+5. [run-container](#run-container)  
+5. [test-container](#test-container)  
+  
+## pre-requisites  
+  
+1. trained model
+2. [source-to-image (s2i)](https://github.com/openshift/source-to-image)  
+3. [Httpie](https://httpie.org/) (Required to run `make test`)   
+  
+## predict     
+ Create a predict function that takes a message object as function parameter containing data instances to predict against   
+and return list of predictions   
```

Comment:
Please add a link to the doc where we describe the predict API https://github.com/CognitiveScale/certifai-reference-models/blob/gettingStartedGuide/docs/instructions.md#model-predict-requestresponse

Reply:
added